geckomat/change_model/makeEcModel.m

```diff
@@ -8,16 +8,18 @@
 %   model.
 %
 % Input:
-%   model       a model in RAVEN format
-%   geckoLight  true if a simplified GECKO light model should be generated.
-%               Optional, default is false.
+%   model        a model in RAVEN format
+%   modelAdapter A modelAdapter, an instance of a class inheriting the 
+%                ModelAdapter class, e.g., YeastAdapter.
+%   geckoLight   true if a simplified GECKO light model should be generated.
```

Comment:
I think it will work automatically? Or do you mean just in the  comment?

Reply:
I think this will work automatically, or do you mean just in the comment?

Reply:
No, currently, `1 `does not work
![Screen Shot 2022-11-12 at 9 53 30 AM](https://user-images.githubusercontent.com/32157802/201466736-81873eb7-c14b-430b-8863-224f3b78f65c.png)

Reply:
When looking at the code, I actually think it should be as it is, that we enforce to use a boolean (logical). Ed explicitly made an effort here :). I know that people in MATLAB often use 1 or 0 (and also in c code), but I think that is a bad habit, true or false is in my mind more clear. But if people think it is a problem, we can remove it, this is not important to me. What do you think @edkerk, do you have some kind of general strategy for this (since you added the check)?

Reply:
`1` and `0` as `true` and `false` should indeed be allowed, will modify the check and consistently apply this whenever a logical input is expected.